### ISSUE TITLE
We're packaging multiple copies of the same gem in our package

### ISSUE THREAD

**Issue by tyler-ball** [chef/chef-workstation#167]
Stumbled across this
```
/opt/chef-workstation/embedded/lib/ruby/gems/2.5.0/gems/chef-run-0.1.114/lib/chef-run/action/converge_target.rb
/opt/chef-workstation/embedded/lib/ruby/gems/2.5.0/gems/chef-cli-0.1.114/lib/chef-cli/action/converge_target.rb
/opt/chef-workstation/embedded/lib/ruby/gems/2.5.0/gems/chef-run-0.1.119/lib/chef-run/action/converge_target.rb
/opt/chef-workstation/embedded/lib/ruby/gems/2.5.0/gems/chef-cli-0.1.103/lib/chef-cli/action/converge_target.rb
/opt/chef-workstation/embedded/lib/ruby/gems/2.5.0/gems/chef-cli-0.1.119/lib/chef-cli/action/converge_target.rb
/opt/chef-workstation/embedded/lib/ruby/gems/2.5.0/gems/chef-run-0.1.103/lib/chef-run/action/converge_target.rb
```

**Comment by robbkidd** [comment]
A cache clearing problem in the omnibus build, mebbe?

**Comment by tyler-ball** [comment]
This looks to be fixed in in the latest release (0.4.2). We also renamed the gem to `chef-apply`
